runtime/doc/autocmd.txt

```diff
@@ -859,6 +860,10 @@ TabEnter			Just after entering a tab page. |tab-page|
 TabLeave			Just before leaving a tab page. |tab-page|
 				A WinLeave event will have been triggered
 				first.
+							*TabNew*
+TabNew                          Before entering a new tab page. |tab-page|
+                                After WinEnter and before TabEnter.
```

Comment:
One thing that drives me crazy about the existing autocmd documentation is that it is not clear when "new" events happen in relation to "enter" events. So it would be great to start documenting this. Does `TabNew` happen *before* or *after* `TabEnter`?